openshift/machine-config-operator
# Config Drift Monitor should alert on unexpected SSH key changes

**- What I did**

The Config Drift Monitor should listen for mutations to the SSH key files. Additionally, in the case of RHCOS 8 -> RHCOS 9, it should alert if an SSH key was written to the old path.

**- How to verify it**

1. Run the Config Drift Monitor unit test suite.
2. Launch a cluster and mutate the SSH key file (`/home/core/.ssh/authorized_keys` for RHCOS 8, `/home/core/.ssh/authorized_keys.d/ignition` for RHCOS 9). The node should drift.
3. Put an SSH key in the incorrect place.

**- Description for the changelog**
Config Drift Monitor watches for mutations to SSH key files
